streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -114,7 +113,7 @@ public StreamsMetricsImpl(final Metrics metrics, final String threadName, final
     }
 
     private static Version parseBuiltInMetricsVersion(final String builtInMetricsVersion) {
-        if (builtInMetricsVersion.equals(StreamsConfig.METRICS_LATEST)) {
+        if (builtInMetricsVersion.equals("latest")) {
```

Comment:
I'm confused by this, we removed the `StreamsConfig.METRICS_LATEST` config, but we've replaced it with a literal string value of the same thing.

EDIT: NM re-reading the PR I think I get it now